transparent-proxy/config/config.go

```diff
@@ -313,7 +326,7 @@ func MergeConfigWithDefaults(cfg Config) Config {
 
 	// .Log
 	result.Log.Enabled = cfg.Log.Enabled
-	if result.Log.Level != DebugLogLevel {
+	if cfg.Log.Level != DebugLogLevel {
```

Comment:
is this change needed - it's the same, you just assigned to it right?

Reply:
It's needed if you configure it and want to customize it.